src/style.css

```diff
@@ -0,0 +1,152 @@
+body {
+  font-weight: 300;
+
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.bar1 {
+  display: block;
+  position: relative;
+  outline: none;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7e1;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+
+}
+
+.search {
+  position: absolute;
+  border: none;
+  outline: none;
+  padding: 0;
+  top: 24px;
+  bottom: 24px;
+  left: 62px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.bar1:hover {
+  display: block;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+
+}
+
+.bar1:focus {
+  display: block;
+  box-sizing: border-box;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+}
+
+.search::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.bar2 {
+  display: block;
+  box-sizing: border-box;
+  outline: none;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e1e1;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 42px;
+  margin-top: 20px;
```

Comment:
Try not duplicate yourself
![image](https://user-images.githubusercontent.com/96409563/225236173-7e74e819-2d09-4af8-acc8-d9749df9543e.png)